src/NuGet.Services.Staging.Manager/Controllers/PackageController.cs

```diff
@@ -143,9 +150,12 @@ public class PackageController : Controller
                     : (IActionResult)new StatusCodeResult((int)HttpStatusCode.Created);
             }
         }
-        private int GetUserKey()
+
+    
+
+        private UserInformation GetUserInformation()
         {
-            return int.Parse(HttpContext.User.Identity.Name);
+            return (UserInformation)HttpContext.Items[Constants.UserInformationKey];

```

Comment:
yes, but since the entire controller is under [Authorize] attribute, this shouldnt be a problem

---
In reply to: [64699080](https://github.com/NuGet/NuGet.Staging/pull/50#discussion_r64699080) [](ancestors = 64699080)